entries/o/octopus.energy.json

```diff
@@ -0,0 +1,13 @@
+{
+  "Octopus Energy": {
+    "domain": "octopus.energy",
+    "url": "octopus.energy",
```

Comment:
If the `uri` field is given, it must contain a valid URI (e.g. start with `https://`). However, if the value is `https://`+domain, it should be left out.
```suggestion
```